IFAS/API/IFAPI.py

```diff
@@ -0,0 +1,48 @@
+import json
+from flask import request, render_template, make_response
+from BL import IFBL
+from flask_restful import Resource
+from Constants import headers
+from logger import logger
+
+class homePage(Resource):
```

Comment:
SRP - What this class is responsible for? you should be clear when you choose names